apps/showcase/src/app/app.module.ts

```diff
@@ -110,7 +111,8 @@ export function registerCustomComponents(): Map<string, any> {
     StylingDevtoolsModule,
     LocalizationDevtoolsModule,
     ConfigurationDevtoolsModule,
-    MonacoEditorModule.forRoot()
+    MonacoEditorModule.forRoot(),
+    MarkdownModule.forRoot()
```

Comment:
can't it be provided in the training route only, to have it lazy loaded ?

Reply:
actually with standalone component I can just remove this import